--- a/app/config/features.neon
+++ b/app/config/features.neon
@@ -15,14 +15,14 @@
 
 # apcupsd service
 apcupsd:
-	enabled: false
+	enabled: true
 # Show documentation
 docs:
 	enabled: true
-	url: https://docs.iqrf.org/iqrf-gateway/
+	url: https://docs.iqrf.org/industrial/
 # Default gateway user password change
 gatewayPass:
-	enabled: false
+	enabled: true
 	user: root
 # Grafana dashboard
 grafana:
@@ -30,7 +30,7 @@
 	url: /grafana/
 # IQRF Gateway Controller configuration
 iqrfGatewayController:
-	enabled: false
+	enabled: true
 # IQRF Gateway Translator configuration
 iqrfGatewayTranslator:
 	enabled: false
@@ -43,36 +43,36 @@
 	path: /etc/systemd/journald.conf
 # Mender client configuration
 mender:
-	enabled: false
+	enabled: true
 # Monitoring service
 monit:
-	enabled: false
+	enabled: true
 # Network manager
 networkManager:
-	enabled: false
+	enabled: true
 # Node-RED
 nodeRed:
-	enabled: false
+	enabled: true
 	url: /node-red/
 # Remount endpoints
 remount:
 	enabled: false
 # SSH daemon manager
 ssh:
-	enabled: false
+	enabled: true
 # Supervisord dashboard
 supervisord:
 	enabled: false
 	url: /supervisord/
 # IQRF TR native upload
 trUpload:
-	enabled: false
+	enabled: true
 # System updater
 updater:
 	enabled: false
 # Unattended upgrades
 unattendedUpgrades:
-	enabled: false
+	enabled: true
 # IQRF Gateway Webapp version checker
 versionChecker:
 	enabled: false
