Adds dependency on IQRF Gateway Uploader
--- a/debian/control
+++ b/debian/control
@@ -14,7 +14,9 @@
 Package: iqrf-gateway-webapp
 Architecture: all
 Depends: ${misc:Depends}, apt, gawk, iproute2,
- iqrf-gateway-daemon, iqrf-journal-reader, net-tools, openssl, passwd,
+ iqrf-gateway-controller, iqrf-gateway-daemon, iqrf-journal-reader,
+ mender-client, mender-connect, modemmanager, monit, net-tools, network-manager,
+ openssh-client, openssl, passwd,
  php8.1-common | php8.2-common,
  php8.1-cli | php8.2-cli,
  php8.1-curl | php8.2-curl,
@@ -25,6 +27,6 @@
  php8.1-sqlite3 | php8.2-sqlite3,
  php8.1-xml | php8.2-xml,
  php8.1-zip | php8.2-zip,
- procps, nginx | apache2, sudo, systemd, systemd-sysv
+ procps, nginx | apache2, sudo, systemd, systemd-sysv, wireguard-tools
 Description: Web user interface for IQRF Gateway Daemon
  IQRF Gateway webapp is an PHP web application for IQRF Gateway configuration.
