Fixes directory paths in Kernel.php
--- a/app/Kernel.php
+++ b/app/Kernel.php
@@ -41,13 +41,14 @@
 	 */
 	public static function boot(): Configurator {
 		$configurator = new Configurator();
-		$configurator->enableTracy(__DIR__ . '/../log');
+		$configurator->setDebugMode(false);
+		$configurator->enableTracy('/var/log/iqrf-gateway-webapp');
 		$configurator->setTimeZone('Europe/Prague');
-		$tempDir = __DIR__ . '/../temp';
+		$tempDir = '/var/cache/iqrf-gateway-webapp';
 		$configurator->setTempDirectory($tempDir);
 		FileSystem::createDir($tempDir . '/sessions');
 		$configurator->createRobotLoader()->addDirectory(__DIR__)->register();
-		$confDir = __DIR__ . '/config';
+		$confDir = '/etc/iqrf-gateway-webapp';
 		$configurator->addConfig($confDir . '/config.neon');
 		try {
 			$version = Json::decode(FileSystem::read(__DIR__ . '/../version.json'));
