Fixes SQLite DB path
--- a/app/config/config.neon
+++ b/app/config/config.neon
@@ -52,7 +52,7 @@
 			# Path to directory containing IQRF Gateway Webapp configuration
 			configDir: '%confDir%'
 			# Path to IQRF Gateway Webapp database
-			database: '%appDir%/config/database.db'
+			database: '/var/lib/iqrf-gateway-webapp/database.db'
 			# Path to directory containing IQRF Gateway Webapp logs
 			logDir: '%appDir%/../log/'
 			#
