Fixes configuration directory path
--- a/app/ConsoleModule/Models/FeatureManager.php
+++ b/app/ConsoleModule/Models/FeatureManager.php
@@ -35,7 +35,7 @@
 	/**
 	 * Path to the webapp's configuration file
 	 */
-	private const CONF_PATH = __DIR__ . '/../../config/features.neon';
+	private const CONF_PATH = '/etc/iqrf-gateway-webapp/features.neon';
 
 	/**
 	 * @var ITranslator Translator
--- a/app/Kernel.php
+++ b/app/Kernel.php
@@ -37,7 +37,7 @@
 		$configurator->setTimeZone('Europe/Prague');
 		$configurator->setTempDirectory('/var/cache/iqrf-gateway-webapp');
 		$configurator->createRobotLoader()->addDirectory(__DIR__)->register();
-		$configurator->addConfig(__DIR__ . '/config/config.neon');
+		$configurator->addConfig('/etc/iqrf-gateway-webapp/config.neon');
 		return $configurator;
 	}
