Fixes log directory path
Index: iqrf-gateway-webapp/app/Kernel.php
===================================================================
--- iqrf-gateway-webapp.orig/app/Kernel.php
+++ iqrf-gateway-webapp/app/Kernel.php
@@ -33,7 +33,7 @@ class Kernel {
 	 */
 	public static function boot(): Configurator {
 		$configurator = new Configurator();
-		$configurator->enableTracy(__DIR__ . '/../log');
+		$configurator->enableTracy('/var/log/iqrf-gateway-webapp');
 		$configurator->setTimeZone('Europe/Prague');
 		$configurator->setTempDirectory('/var/cache/iqrf-gateway-webapp');
 		$configurator->createRobotLoader()->addDirectory(__DIR__)->register();
