Fixes cache directory path
Index: iqrf-gateway-webapp/app/Kernel.php
===================================================================
--- iqrf-gateway-webapp.orig/app/Kernel.php
+++ iqrf-gateway-webapp/app/Kernel.php
@@ -39,7 +39,7 @@
 		$configurator = new Configurator();
 		$configurator->enableTracy(__DIR__ . '/../log');
 		$configurator->setTimeZone('Europe/Prague');
-		$configurator->setTempDirectory(__DIR__ . '/../temp');
+		$configurator->setTempDirectory('/var/cache/iqrf-gateway-webapp');
 		$configurator->createRobotLoader()->addDirectory(__DIR__)->register();
 		$configurator->addConfig(__DIR__ . '/config/config.neon');
 		$version = Json::decode(FileSystem::read(__DIR__ . '/../version.json'));
