--- a/debian/control
+++ b/debian/control
@@ -15,7 +15,7 @@
 Architecture: all
 Depends: ${misc:Depends}, apt, gawk, iproute2,
  iqrf-gateway-controller, iqrf-gateway-daemon, iqrf-journal-reader,
- mender-client, mender-connect, modemmanager, monit, net-tools, network-manager,
+ mender-client, mender-connect, modemmanager, net-tools, network-manager,
  openssh-client, openssl, passwd,
  php7.4-common | php8.0-common | php8.1-common,
  php7.4-cli | php8.0-cli | php8.1-cli,
