--- a/app/config/features.neon
+++ b/app/config/features.neon
@@ -19,14 +19,14 @@
 # Show documentation
 docs:
 	enabled: true
-	url: https://docs.iqrf.org/iqube/
+	url: https://www.iqaros.eu/installation.html
 # Default gateway user password change
 gatewayPass:
 	enabled: false
 	user: root
 # Grafana dashboard
 grafana:
-	enabled: false
+	enabled: true
 	url: /grafana/
 # IQRF Gateway Controller configuration
 iqrfGatewayController:
@@ -39,13 +39,13 @@
 	enabled: false
 # iTemp service
 iTemp:
-	enabled: false
+	enabled: true
 # Mender client configuration
 mender:
 	enabled: false
 # Monitoring service
 monit:
-	enabled: true
+	enabled: false
 # Network manager
 networkManager:
 	enabled: true
