--- a/app/config/features.neon
+++ b/app/config/features.neon
@@ -16,13 +16,13 @@
 # Show documentation
 docs:
 	enabled: true
-	url: https://docs.iqrf.org/iqube/
+	url: https://www.iqaros.eu/installation.html
 # Default gateway user password change
 gatewayPass:
 	enabled: false
 	user: root
 grafana:
-	enabled: false
+	enabled: true
 	url: /grafana/
 # IQRF Gateway Controller configuration
 iqrfGatewayController:
@@ -35,19 +35,19 @@ iqrfRepository:
 	enabled: false
 # iTemp service
 iTemp:
-	enabled: false
+	enabled: true
 # Mender client configuration
 mender:
 	enabled: false
 # Monitoring service
 monit:
-	enabled: true
+	enabled: false
 # Network manager
 networkManager:
 	enabled: true
 # Node-RED
 nodeRed:
-	enabled: false
+	enabled: true
 	url: /node-red/
 ntp:
 	enabled: false
