Set theme for the IQAROS
--- a/.env
+++ b/.env
@@ -1,2 +1,3 @@
 VUE_APP_BASE_URL=/system/
 VUE_APP_CYPRESS_ENABLED=0
+VUE_APP_THEME=iqaros
--- a/src/main.ts
+++ b/src/main.ts
@@ -34,7 +34,7 @@
 
 import App from './App.vue';
 
-import './css/themes/generic.scss';
+import './css/themes/iqaros.scss';
 import './css/app.scss';
 import 'vue-datetime/dist/vue-datetime.css';
 import 'vue-select/dist/vue-select.css';
