Set theme for the IQAROS
--- a/packages/frontend/.env
+++ b/packages/frontend/.env
@@ -15,4 +15,4 @@
 
 VITE_BASE_URL=/system/
 VITE_CYPRESS_ENABLED=0
-VITE_THEME=generic
+VITE_THEME=iqaros
--- a/packages/frontend/src/main.ts
+++ b/packages/frontend/src/main.ts
@@ -21,7 +21,7 @@
 import ThemeManager from './helpers/themeManager';
 import App from '@/App.vue';
 
-import './styles/themes/generic.scss';
+import './styles/themes/iqaros.scss';
 import './styles/app.scss';
 import 'vue-datetime/dist/vue-datetime.css';
 import 'vue-select/dist/vue-select.css';
