--- a/app/config/features.neon
+++ b/app/config/features.neon
@@ -16,7 +16,7 @@
 # Show documentation
 docs:
 	enabled: true
-	url: https://docs.iqrf.org/iqrf-gateway/
+	url: https://docs.iqrf.org/iqube/
 # Default gateway user password change
 gatewayPass:
 	enabled: false
@@ -26,7 +26,7 @@ grafana:
 	url: /grafana/
 # IQRF Gateway Controller configuration
 iqrfGatewayController:
-	enabled: false
+	enabled: true
 # IQRF Gateway Translator configuration
 iqrfGatewayTranslator:
 	enabled: false
@@ -42,13 +42,13 @@ journal:
 	path: /etc/systemd/journald.conf
 # Mender client configuration
 mender:
-	enabled: false
+	enabled: true
 # Monitoring service
 monit:
-	enabled: false
+	enabled: true
 # Network manager
 networkManager:
-	enabled: false
+	enabled: true
 # Node-RED
 nodeRed:
 	enabled: false
@@ -58,7 +58,7 @@ remount:
 	enabled: false
 # SSH daemon manager
 ssh:
-	enabled: false
+	enabled: true
 # Supervisord dashboard
 supervisord:
 	enabled: false
@@ -68,10 +68,10 @@ updater:
 	enabled: false
 # IQRF TR native upload
 trUpload:
-	enabled: false
+	enabled: true
 # Unattended upgrades
 unattendedUpgrades:
-	enabled: false
+	enabled: true
 # IQRF Gateway Webapp version checker
 versionChecker:
 	enabled: false
