--- a/app/config/features.neon
+++ b/app/config/features.neon
@@ -1,13 +1,13 @@
 # Show documentation
 docs:
 	enabled: true
-	url: https://docs.iqrf.org/iqrf-gateway/
+	url: https://docs.iqrf.org/iqube/
 grafana:
 	enabled: false
 	url: /grafana/
 # IQRF Gateway Controller configuration
 iqrfGatewayController:
-	enabled: false
+	enabled: true
 # IQRF Gateway Translator configuration
 iqrfGatewayTranslator:
 	enabled: false
@@ -23,13 +23,13 @@ nodeRed:
 	url: /node-red/
 # PIXLA mamagement system
 pixla:
-	enabled: false
+	enabled: true
 # Root password change
 rootpass:
 	enabled: false
 # SSH daemon manager
 ssh:
-	enabled: false
+	enabled: true
 # Supervisord dashboard
 supervisord:
 	enabled: false
@@ -42,10 +42,10 @@ updater:
 	enabled: false
 # IQRF TR native upload
 trUpload:
-	enabled: false
+	enabled: true
 # Unattended upgrades
 unattendedUpgrades:
-	enabled: false
+	enabled: true
 # IQRF Gateway Webapp version checker
 versionChecker:
 	enabled: false
