--- a/app/config/features.neon
+++ b/app/config/features.neon
@@ -15,11 +15,11 @@
 
 # apcupsd service
 apcupsd:
-	enabled: false
+	enabled: true
 # Show documentation
 docs:
 	enabled: true
-	url: https://docs.iqrf.org/iqrf-gateway/
+	url: https://docs.iqrf.org/iqube/
 # Default gateway user password change
 gatewayPass:
 	enabled: false
@@ -30,7 +30,7 @@
 	url: /grafana/
 # IQRF Gateway Controller configuration
 iqrfGatewayController:
-	enabled: false
+	enabled: true
 # IQRF Gateway Translator configuration
 iqrfGatewayTranslator:
 	enabled: false
@@ -46,13 +46,13 @@
 	path: /etc/systemd/journald.conf
 # Mender client configuration
 mender:
-	enabled: false
+	enabled: true
 # Monitoring service
 monit:
-	enabled: false
+	enabled: true
 # Network manager
 networkManager:
-	enabled: false
+	enabled: true
 # Node-RED
 nodeRed:
 	enabled: false
@@ -62,20 +62,20 @@
 	enabled: false
 # SSH daemon manager
 ssh:
-	enabled: false
+	enabled: true
 # Supervisord dashboard
 supervisord:
 	enabled: false
 	url: /supervisord/
 # IQRF TR native upload
 trUpload:
-	enabled: false
+	enabled: true
 # System updater
 updater:
 	enabled: false
 # Unattended upgrades
 unattendedUpgrades:
-	enabled: false
+	enabled: true
 # IQRF Gateway Webapp version checker
 versionChecker:
 	enabled: false
