--- a/app/config/features.neon
+++ b/app/config/features.neon
@@ -1,21 +1,21 @@
 parameters:
-	docs: https://docs.iqrf.org/iqrf-gateway/
+	docs: https://docs.iqrf.org/iqd-gw-01/
 	features:
 		# Show documentation
 		docs: true
 		# IQRF TR native upload
-		trUpload: false
+		trUpload: true
 		# Node-RED
 		node-red: false
 		# PIXLA mamagement system
-		pixla: false
+		pixla: true
 		# Network manager
 		networkManager: false
 		# SSH daemon manager
-		ssh: false
+		ssh: true
 		# Supervisord dashboard
 		supervisord: false
 		# System updater
 		updater: false
 		# Unattended upgrades
-		unattendedUpgrades: false
+		unattendedUpgrades: true
